api.planx.uk/.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "rules": {
+    "@typescript-eslint/no-non-null-assertion": "warn",
+  },
```

Comment:
nit / opinion:  Could we add the following rule here?

```json
"@typescript-eslint/no-unused-vars": ["warn", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_"  }]
```

This follows the pattern we're currently using and I personally find it sometimes to be a bit more explicit than just commas. Not a strongly held opinion!